### PR TITLE
Articulation points editorial

### PR DESCRIPTION
**Pull request by brebenelmihnea**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Review by SansPapyrus683 — APPROVED** [review]
yeah this is nice

File: content/6_Advanced/BCC_2CC.mdx [hunk 1]
```diff
@@ -348,19 +348,87 @@ int main() {
...
+			g[y].push_back(x);
+		}
+
+		dfs(0, -1);
+
+		int cnt = 0;
+		for (int i = 0; i < n; i++) {
+			if (ap[i]) { cnt++; }
+		}
+		cout << cnt << '\n';
```
Comment (SansPapyrus683) [inline content/6_Advanced/BCC_2CC.mdx:425]: rename `cnt` to something more informative

Comment (brebenelmihnea) [reply content/6_Advanced/BCC_2CC.mdx]: Ok

**Comment by SansPapyrus683** [comment]
image

p sure these should have a focus problem format?

**Comment by brebenelmihnea** [comment]
> Preview

@SansPapyrus683 I made them focus problems.

**Comment by SansPapyrus683** [comment]
image

uh i don't see it on my end

**Comment by brebenelmihnea** [comment]
Ok. I thought you meant just the problems in sh. I made that too focus.

**Review by ryanchou-dev — APPROVED** [review]
firee

**Pull request merged**